grep.js

```diff
@@ -21,18 +19,16 @@ const pickTests = (filename, foundTests, cypressConfig) => {
 
   const fgrep = cypressConfig.env.fgrep
   const grep = cypressConfig.env.grep // assume string for now, not regexp
-  if (!fgrep && !grep) {
-    // run all tests
-    return foundTests
-  }
 
   if (fgrep) {
+    console.log("\tJust tests tagged with: %s", grep)
     if (!filename.includes(fgrep)) {
-      console.log('spec filename %s not matching fgrep "%s"', filename, fgrep)
+      console.warn('\tTest filename %s did not match fgrep "%s"', filename, fgrep)
       return
     }
   }
   if (grep) {
+    console.log("\tJust tests tagged with: %s", grep)
```

Comment:
We are using grep, it makes more sense to print the grep than the full filepath of each file. And it looks nice with the default reporter of Cypress.